lasagne/layers/conv.py

```diff
@@ -125,9 +125,7 @@ def conv_input_length(output_length, filter_size, stride, pad=0):
     int or None
         The smallest input size corresponding to the given convolution
         parameters for the given output size, or ``None`` if `output_size` is
-        ``None``. For a strided convolution, any input size of up to
-        ``stride - 1`` elements larger than returned will still give the same
-        output size.
+        ``None``.
```

Comment:
Why did you remove this? The functionality of `conv_input_length` stays the same.